endpoints/getting-started/main.py

```diff
@@ -53,7 +53,7 @@ def echo():
 
 # [START endpoints_auth_info_backend]
 def auth_info():
-    """Retrieves the authenication information from Google Cloud Endpoints."""
+    """Retrieves the authentication information from Google Cloud Endpoints."""
```

Comment:
```suggestion
    """Retrieves the authenication information from Google Cloud Endpoints."""
```
removing because this is fixed in #11332